doc/distrib/NodeHelpFiles/_review/Revit.Elements.Views.View.AddFilter.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`View.AddFilter` adds the given filter to the given view. If a view has a view template applied, `View.AddFilter` will add the filter to the view's view template.
+
```

Comment:
This sentence has too many "Views" and my brain can't handle (lol).  Maybe rephrase as "View.AddFilter adds the specified filter to the given view. If the view has a view template applied, this node will add the filter to it's view template."